acceptance/config/nodes/win2012r2-rubyx64.yaml

```diff
@@ -12,6 +21,13 @@ HOSTS:
     ruby_arch: x64
     hypervisor: vcloud
     template: win-2012r2-x86_64
+    puppetconfdir: C:/ProgramData/PuppetLabs/puppet/etc
+    puppetcodedir: C:/ProgramData/PuppetLabs/puppet/etc
+    puppetvardir: C:/ProgramData/PuppetLabs/puppet/var
+    distmoduledir: C:/ProgramData/PuppetLabs/puppet/etc/modules
+    # puppet-agent 0.3.1 package on the master still uses /usr/share
+    sitemoduledir: C:/usr/share/puppet/modules
+    hieraconf: C:/ProgramData/PuppetLabs/puppet/etc/hiera.yaml
```

Comment:
Currently, AIO code on windows does the following:

    distmoduledir: C:/ProgramData/PuppetLabs/puppet/etc/code/modules
    sitemoduledir: C:/ProgramData/PuppetLabs/puppet/var/modules
    hieraconf: C:/ProgramData/PuppetLabs/puppet/etc/code/hiera.yaml

The distmoduledir and hieraconf will probably need to move under $codedir. Probably best to make that change all at once, in run_mode, specs, install.rb, and acceptance node configs.